src/app/core/entity-components/entity-utils/dynamic-form-components/edit-component.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { EditComponent } from "./edit-component";
+import { FormControl, FormGroup } from "@angular/forms";
+
+export const FORM_CONTROL_NAME = "testProperty";
+
+/**
+ * A simple helper class that sets up a EditComponent with the required FormGroup
+ * @param component that extends EditComponent
+ */
+export function setupEditComponent<T>(component: EditComponent<T>): FormGroup {
```

Comment:
can we make the FORM_CONTROL_NAME as a parameter of this function? Would make it a bit more clear (no depending on the other constant) and flexible - although in line 12 currently the name is hardcoded?

not super essential, just an idea. Otherwise maybe rename to `setupEditComponentForTestProperty`?